### ISSUE TITLE
High fan speed

### ISSUE THREAD

**Issue by bcordoba555** [featurecat/lizzie#698]
I am noticing very high fan noise while using Lizzie yet the CPU usage is very low (10%), whereas using Folding@home the CPU usage is very high (90%) but fan noise is very quiet.  Any ideas on how to fix that?

Thank you very much for making a great program available for free!  It's very useful to my game.

Benoit

**Comment by featurecat** [comment]
Perhaps it's because Leela Zero uses the gpu which could be heating your computer up more than the cpu?

**Comment by featurecat** [comment]
I can't think of a way to fix it though... You could try using cpu only mode but then the AI would not be as fast.

**Comment by bcordoba555** [comment]
Yes, it could be.  I posted on the Leela Zero Github as well but no response yet.  Since i am a non-programmer user it is still difficult even with the Windows distribution to change settings.  In any case it is not a huge problem.  Thank you for responding.
